connexion/app.py

```diff
@@ -129,6 +132,7 @@ def add_api(self, swagger_file, base_path=None, arguments=None, auth_all_paths=N
                   swagger_url=swagger_url,
                   resolver=resolver,
                   validate_responses=validate_responses,
+                  security_decorator=security_decorator,
```

Comment:
If you pass the security_decorator variable into the constructor of `App` class it should be considered here. Just like the other global configs like `swagger_ui`, `swagger_path`, etc.

Precisely, I think you should add something like this line:
```python
security_decorator = security_decorator if security_decorator is not None else self.security_decorator
```